src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -8,7 +8,7 @@
       1591: Documentation warnings
       NU5100: Warning that gets triggered because a .dll is not placed under lib folder on package. This is by design as we want AutoML Interactive to be under interactive-extensions folder.
        -->
-    <NoWarn>$(NoWarn);1591;NU5100</NoWarn>
+    <NoWarn>$(NoWarn);1591;NU5100s</NoWarn>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);AddAutoMLInteractiveToInteractiveExtensionsFolder</TargetsForTfmSpecificContentInPackage>
```

Comment:
is it intentional to have the `s` at the end in `NU5100s`? do we need to update the comment above?

Reply:
Seeing failure like:

```
 error NU5100: Warning As Error: The assembly 'interactive-extensions\dotnet\Microsoft.ML.AutoML.Interactive.dll' is not inside the 'lib' folder and hence it won't be added as a reference when the package is installed into a project. Move it into the 'lib' folder if it needs to be referenced. [D:\a\_work\1\s\src\Microsoft.ML.AutoML\Microsoft.ML.AutoML.csproj]
```

Reply:
Not at all. I must have fat fingered that somehow.